model/map/src/main/java/org/keycloak/models/map/client/MapProtocolMapperEntity.java

```diff
@@ -27,10 +28,7 @@
  */
 @GenerateEntityImplementations
 @DeepCloner.Root
-public interface MapProtocolMapperEntity extends UpdatableEntity {
-
-    String getId();
-    void setId(String id);
+public interface MapProtocolMapperEntity extends UpdatableEntity, AbstractEntity {
```

Comment:
I made MapProtocolMapperEntity interface extend AbstractEntity. It has method get/setId anyway. Do you see any reason why we don't want this?